src/effects/lv2/lv2manifest.cpp

```diff
@@ -189,8 +189,8 @@ void LV2Manifest::buildEnumerationOptions(const LilvPort* port,
         const LilvNode* description = lilv_scale_point_get_label(option);
         const LilvNode* value = lilv_scale_point_get_value(option);
         QString strDescription(lilv_node_as_string(description));
-        param->appendStep(qMakePair(strDescription, 
-		(double)lilv_node_as_float(value)));
+        param->appendStep(qMakePair(strDescription,
+                (double)lilv_node_as_float(value)));
```

Comment:
Use a static_cast instead of C style casts.

Reply:
this is clang-format reformatting, but sure I can change that too